tags/tags.toml

```
@@ -1181,3 +1181,12 @@ content = """
 To get multiple images to display within one embed, use multiple embeds within the same message that use the same URL `.setURL(x)`/`url: x`.
 *Note that this is meant for discord-side embeds and may break without notice.*
 """
+[invite-tracker]
+keywords = ["track-invite", "used-invite", "invite-used"]
```

```suggestion
keywords = ["tracking-invites", "used-invite", "who-invited-a-user"]
```
prob better keywords